docs/components/operate/userguide/updating-operate.md

```diff
@@ -4,6 +4,8 @@ title: Updating Operate
 description: Instructions on how to update your Operate
 ---
 
+<!--- Sooooo we have this file but I don't see it in the sidebar. Should this be added? --->
```

Comment:
I would vote to delete it. If we do keep it, this is only relevant for SM customers and should be moved.

Reply:
LOL. Did some digging and found this content _is_ in SM and was never removed here. Deleting 🥳